File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/save-vetting-batches_param.ts


import { IVettingBatch_Item } from './ivetting-batch_item';
import { ISaveVettingBatches_Param } from './isave-vetting-batches_param';

export class SaveVettingBatches_Param implements ISaveVettingBatches_Param {
  
	public Batches?: IVettingBatch_Item[];
  
	public ErrorMessages: string[] = null;
}

File: GTTS/INL.VettingService.Models/SaveVettingBatches_Param.cs
using INL.Services.Models;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public class SaveVettingBatches_Param : BaseParam, ISaveVettingBatches_Param
    {
        public SaveVettingBatches_Param() : base() { }
        public List<IVettingBatch_Item> Batches { get; set; }
    }
}
